src/Symfony/Component/HttpClient/HttplugClient.php

```diff
@@ -57,7 +57,7 @@
  *
  * @author Nicolas Grekas <p@tchwork.com>
  */
-final class HttplugClient implements HttplugInterface, HttpAsyncClient, RequestFactory, StreamFactory, UriFactory, ResetInterface
+final class HttplugClient implements HttplugInterface, HttpAsyncClient, RequestFactoryInterface, StreamFactoryInterface, UriFactoryInterface, RequestFactory, StreamFactory, UriFactory, ResetInterface
```

Comment:
shouldn't we add a check for `RequestFactoryInterface` before the class definition, as done for RequestFactory ?

Reply:
good ideas, updated